pkg/ioutil/pagewriter.go

```diff
@@ -95,12 +95,23 @@ func (pw *PageWriter) Write(p []byte) (n int, err error) {
 	return n, werr
 }
 
+// Flush flushes buffered data.
 func (pw *PageWriter) Flush() error {
+	_, err := pw.flush()
+	return err
+}
```

Comment:
Should `Flush()` be deprecated or even removed? Otherwise, the metrics where `FlushN` captured isn't complete?

Reply:
It was an external package, so there could be other projects importing this method. So we rather keep it.